resources/install/debian/postinst

```diff
@@ -116,7 +116,7 @@ case "$1" in
             echo "org.jitsi.videobridge.xmpp.user.shard.MUC_NICKNAME=$(uuidgen)" >> $NEW_JITSI_CONFIG
         fi
 
-        if ! grep -q "--apis=," "$CONFIG" ;then
+        if ! grep -q "\--apis=," "$CONFIG" ;then
```

Comment:
`grep -q -e "--apis=,"` would also work -- -e means "interpret the next arg as the pattern, even if it starts with a dash."